homeassistant/components/media_player/mpd.py

```diff
@@ -202,12 +201,23 @@ def media_album_name(self):
     @property
     def volume_level(self):
         """Return the volume level."""
-        return int(self._status['volume'])/100
+        if 'volume' in self._status:
+            return int(self._status['volume'])/100
 
```

Comment:
Return `None` here to return consistently.

Reply:
Fixed.